src/main/java/org/myrobotlab/service/abstracts/AbstractServo.java

```diff
@@ -238,6 +238,13 @@ public AbstractServo(String n, String id) {
     // create our default TimeEncoder
     if (encoder == null) {
       encoder = new TimeEncoder(this);
+      // if the encoder has a current value - we initialize the
+      // servo with that value
+      Double savedPos = encoder.getPos();
+      if (savedPos != null) {
+        log.info("found previous values for {} setting initial position to {}", getName(), savedPos );
```

Comment:
I think we're trying to do too much with the initial position of the servos.  I like  the deterministic behavior as stated by Gael in the shout box.  Saving and attempting to know the current state of a hobby servo after a power cycle really need to be a problem that is out of scope.  Servos do not provide positional feedback, so , i think we're adding complexity to solve a problem that fundamentally isn't solvable with a standard hobby servo.